src/pykeen/losses.py

```diff
@@ -205,6 +212,10 @@ def forward(
     MarginRankingLoss: dict(
         margin=dict(type=int, low=0, high=3, q=1),
     ),
+    NSSALoss: dict(
+        margin=dict(type=int, low=3, high=30, q=3),
+        adversarial_temperature=dict(type='categorical', choices=[0.5, 1.0]),
```

Comment:
@cthoyt I might be better to actually define a uniform float range here, instead of restricting it to exactly those two values.

Reply:
I was torn between these two options. I wonder if there are diminishing returns for doing a search through this space when one of the discrete values might be *good enough*

Reply:
Updated in e89ed12 to be continuous